test/e2e/framework/volume/fixtures.go

```diff
@@ -488,6 +489,13 @@ func runVolumeTesterPod(client clientset.Interface, config TestConfig, podSuffix
 
 	for i, test := range tests {
 		volumeName := fmt.Sprintf("%s-%s-%d", config.Prefix, "volume", i)
+
+		privileged := false
+		if selinux.SELinuxEnabled() && test.Mode != v1.PersistentVolumeBlock {
```

Comment:
please comment why this is necessary - 1. selinux, 2. moby issue (+link)

Reply:
Added comment